.htmltest.yml

```diff
@@ -16,8 +16,6 @@ IgnoreDirs:
   - ^es/docs/concepts/signals/baggage/
   - ^es/docs/zero-code/php/
   # TODO drop next lines after https://github.com/open-telemetry/opentelemetry.io/issues/5555 is fixed for these pages:
-  - ^fr/docs/concepts/glossary/
-  # TODO drop next lines after https://github.com/open-telemetry/opentelemetry.io/issues/5555 is fixed for these pages:
```

Comment:
Oops, my bad, this should have been deleted from `content/fr/docs/what-is-opentelemetry.md`. Let me figure out why the build checks didn't report this.